examples/index.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { HashRouter as Router, Route, NavLink as Link, Switch } from 'react-router-dom';
+import { HashRouter as Router, Route, NavLink as Link, Routes } from 'react-router-dom';
```

Comment:
I followed this guide: https://reactrouter.com/docs/en/v6/upgrading/v5#upgrade-all-switch-elements-to-routes